src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DDMMYYYY = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    public static boolean between(LocalDate d, LocalDate from, LocalDate to) {
+        return (d.isAfter(from) || d.isEqual(from)) && (d.isBefore(to) || d.isEqual(to));
+    }
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int [] salary = new int[names.length];
+        LocalDate from = LocalDate.parse(dateFrom, DDMMYYYY);
+        LocalDate to = LocalDate.parse(dateTo, DDMMYYYY);
+        for (int i = 0; i < data.length; i++) {
+            String[] pices = data[i].split(" ");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
            String[] pices = data[i].split(SEPARATOR);
```